bridges/FlashbackBridge.php

```diff
@@ -0,0 +1,185 @@
+			return 'Thread ' . $thread . ' - Flashback';
+		} elseif ($this->getInput('u')) {
+			$user = $this->getInput('u');
+			return 'User ' . $user . ' - Flashback';
+		} elseif ($this->getInput('s')) {
+			$search = $this->getInput('s');
+			return 'Search: ' . $search . ' - Flashback';
+		}
+
+		return self::NAME;
+	}
+
+	public function collectData()
+	{
+		if ($this->getInput('c')) {
+			$page = self::URI . '/f' . $this->getInput('c');
+		} elseif ($this->getInput('a')) {
+			$page = self::URI . '/find_threads_by_tag.php?tag=' . $this->getInput('a');
+		} elseif ($this->getInput('t')) {
+			$page = self::URI . '/t' . $this->getInput('t');
+			$page = $page . 's'; // last-page
+		} elseif ($this->getInput('u')) {
+			$page = self::URI . '/find_posts_by_user.php?userid=' . $this->getInput('u');
+		} elseif ($this->getInput('s')) {
+			if ($this->getInput('type') == 'posts') {
+				$page = self::URI . '/sok/?query=' . $this->getInput('s') . '&search_post=1&sp=1&so=pd';
+			} else {
+				$page = self::URI . '/sok/?query=' . $this->getInput('s') . '&search_post=0&sp=1&so=pd';
+			}
+		}
+
+		$html = getSimpleHTMLDOM($page);
+
+		if ($this->getInput('c') || $this->getInput('a')) {
+			$category = $this->getInput('c');
+			$array = $html->find('table#threadslist tbody tr');
+			foreach ($array as $key => $element) {
+				$item = array();
+				$item['uri'] = self::URI . $element->find('td.td_title a', 0)->href;
+				$item['title'] = trim(utf8_encode($element->find('td.td_title a', 0)->innertext));
+				$item['author'] = trim(utf8_encode(
+					$element->find('td.td_title span.thread-poster span', 0)->innertext)
+					);
+				$timestamp = $element->find('td.td_last_post div', 0);
+				if (isset($timestamp->plaintext)) {
+					$item['timestamp'] = strtotime(str_replace(array('Ig&aring;r', 'Idag'),
+					array('yesterday', 'today'), trim($timestamp->plaintext)));
+				}
+				$item['content'] = $item['title'] . '<br />' . trim(preg_replace('/\t+/', '',
+					$element->find('td.td_replies', 0)->innertext));
+				$item['uid'] = preg_split('/(\/)/', $element->find('td.td_title a', 0)->href)[1];
+				$this->items[] = $item;
+			}
+		} elseif ($this->getInput('t')) {
+			$tags = $html->find('div.hidden-xs a.tag');
+			$array = $html->find('div.post');
+
+			foreach ($array as $key => $element) {
+				$item = array();
+				$item['uri_post'] = self::URI . $element->find('div.post-heading a', 2)->href;
+				$item['uri'] = self::URI . '/' . preg_split('/(\/s)/', $item['uri_post'])[1] . '#' .
+					preg_split('/(\/s)/', $item['uri_post'])[1];
+				$item['uri_thread'] = $page;
+				$item['author'] = utf8_encode($element->find('div.post-user ul li', 0)->innertext);
+				$item['author_link'] = self::URI . $element->find('div.post-user ul li a', 0)->href;
+				$item['post_nr'] = $element->find('div.post-heading a strong', 0)->innertext;
+				$item['timestamp'] = strtotime(
+					str_replace(
+						array('Ig&aring;r', 'Idag'), array('yesterday', 'today'), \
```

Comment:
This trailing `\` is a syntax error on PHP 8.

```
[Tue Jan  4 22:54:11 2022] [::1]:57496 [200]: GET / - syntax error, unexpected token "\", expecting ")" in rss-bridge/bridges/FlashbackBridge.php on line 141
```

```
$  php --version
PHP 8.0.14 (cli) (built: Dec 17 2021 14:16:47) ( NTS )
Copyright (c) The PHP Group
Zend Engine v4.0.14, Copyright (c) Zend Technologies
```

Reply:
+1, was going to report this as well.

Reply:
Oh, this is already fixed in #2405, never mind.